game/script-farewells.rpy

```diff
@@ -1954,3 +1954,33 @@ label farewell_night_good_night:
         n 3uchbgl "Sweet dreams!"
 
     return { "quit": None }
+
+# Natsuki wishes the player a nice rest of their day
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._farewell_database,
+            label="farewell_rest_of_your_day",
+            unlocked=True,
+                conditional="store.jn_get_current_hour() >= 16 or store.jn_get_current_hour() <= 19",
+                affinity_range=(jn_affinity.NORMAL, jn_affinity.LOVE),
+                ),
+            topic_group=TOPIC_TYPE_FAREWELL
+            )
+
+label farewell_rest_of_your_day:
+    n 2kllajl "Oh, you're heading out,{w=0.4} [player]?"
+    n 2nnmssl "That's alright, {w=0.4}{nw}"
+    extend 2fsqfsl " I guess..."
+    if Natsuki.isEnamored(higher=False):
+        n 1unmbgl "Have a nice rest of your day, [player]!"
+        return { "quit": None }
```

Comment:
You don't need to use `return { "quit": None }` after every condition, one time at the end of your topic is enough as the last one will always be called.